app/controllers/qsets_controller.rb

```diff
@@ -3,17 +3,25 @@ class QsetsController < ApplicationController
 
   # shows all qsets
   def index
-    @qsets = @qsets.order(:name)
-    @question_counts = Question.all.group(:qset_id).count
+    if current_user.org.nil?
+      redirect_to root_url, alert: "Please contact user support at support@askup.net."
+    else
+      redirect_to (current_user.org) 
+    end
   end
 
   # handles the request to show all questions in a qset
   def show
+    @question_counts = Question.all.group(:qset_id).count
     @feedback_active = !!current_user
     @questions = Question.includes(:answers).where(qset_id: @qset.id).order(created_at: :desc)
     @filter_mine = true if cookies[:all_mine_other_filter] == 'mine'
     @filter_other = true if cookies[:all_mine_other_filter] == 'other'
     @filter_all = true unless @filter_mine or @filter_other
+    @qsets = Qset.where(parent_id: @qset.id)
```

Comment:
This can be simplified to
```
@qsets = @qset.children
```
(...or that `@qset.children` can be moved directly into the _qset_list partial (line 2).)